.github/workflows/manual-release-charts.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Manual event to release helm chart to public site on release branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        default: "1.1.0"
+        description: "Release Version"
+        required: true
+        type: string
+      artifacthub:
```

Comment:
please comment or remove this artifacthub section. We will publish to one place only per earlier discussion. Assume you leave it here and then do not publish per discussion, someone will think its a bug.